contracts/contracts/solidity/mocks/MockValidator.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why do we require this mock?

Looks like our tests are testing the `MockValidator` instead of our `StarkNetValidator`.
What we should really test is our `StarkNetValidator` with the mocked/simulated bridge.